### DISCUSSION TITLE
Unable to upload video using FileUpload

### DISCUSSION THREAD

**Discussion by Rahul-codoffer** [filamentphp/filament#11774] [Help]
### Package

Form builder

### Package Version

v2.0.0

### How can we help you?

I'm trying to upload video but getting error from livewire side 
**Error : The files.0 failed to upload.** 

I have tried to upload image and zip and it works fine with image and zip upload only getting error while uploading video,
Even i'm using the only 2MB video.

Can any one help me with this ? I'm using the Filament 2

**Comment by leandrocfe** [comment]
Did you publish the livewire config and change the max param in the rules?
https://laravel-livewire.com/docs/2.x/file-uploads#configuration

**Comment by Rahul-codoffer** [comment]
Yes , i have change the max param , its 20 MB and i'm trying to upload a just 3 MB video
